src/lib/services/feature-toggle-service.ts

```diff
@@ -510,6 +510,29 @@ class FeatureToggleService {
         return this.unprotectedUpdateStrategy(id, updates, context, userName);
     }
 
+    async optionallyDisableFeature(
```

Comment:
this was previously in a controller but it means that change requests calling service methods don't have access to full functionality. Overall controller shouldn't orchestrate application logic, application service should do it.